[BUG]  Removed python files are linted by the format python script

<!-- :warning: Please, try to follow the template -->

### System information

- **RTI Product**:
- **Version**:
- **Operating system**:
- **Compiler**:
- **Compiler version**:
- **Additional information**:

### What is the current behavior?

```
Run base_commit="07a631cec6ed3756f290147984f2585327c36ff8"
Starting format checker...
Checking file changes in the commit range: ['07a631cec6ed3756f290147984f2585327c36ff8', '4b0cafcc2d9b5e613da01b1112f1b17f4c266836'].

==================================== Python ===================================
Checking Python files format with Black: ['resources/ci_cd/linux_install.py', 'resources/static_analysis/run_static_analysis.py']

Usage: black [OPTIONS] SRC ...
Try 'black -h' for help.

Error: Invalid value for 'SRC ...': Path 'resources/static_analysis/run_static_analysis.py' does not exist.

Error: Process completed with exit code 1.
```

### Steps to reproduce the issue

### Expected behavior

### Suggested solutions

--- a/resources/ci_cd/linux_format.py
+++ b/resources/ci_cd/linux_format.py
@@ -136,7 +136,7 @@ def get_git_files(
         command.append("ls-files")
         print("Getting all the repository files.")
     else:
-        command = ["diff", "--no-commit-id", "--name-only", "-r"]
+        command = ["diff", "--no-commit-id", "--name-only", "-r", "--diff-filter=dr"]
 
         if commits:
             command.extend([*commits])

